src/GuiBase/Viewer/Viewer.cpp

```diff
@@ -692,4 +694,11 @@ void Gui::Viewer::enableDebugDraw( int enabled ) {
     m_currentRenderer->enableDebugDraw( enabled );
 }
 
+void Gui::Viewer::update( const Scalar dt ) {
+    if ( m_gizmoManager != nullptr ) { m_gizmoManager->updateValues(); }
```

Comment:
I'm a bit perplex here, is the `GizmoManager` really a time dependent data?

Reply:
It is as its position is time dependent. At least if I understand comments that I removed

Reply:
this comes from the removed lines 420-421  from `BaseApplication::radiumFrame`

> // also update gizmo manager to deal with annimation playing / reset
> m_viewer->getGizmoManager()->updateValues();